<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yandex Map API</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>602</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="inputLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Искать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="fullAddressLine">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mapLabel">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>450</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>450</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Схема</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Спутник</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Гибрид</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Сброс</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
